README.md

```diff
@@ -1,72 +1,93 @@
-5. run `docker-compose up -d` to start the containers
-6. point your browser to [http://io-onboarding-backend:3000](http://io-onboarding-backend:3000)
-
-#### Custom configuration
-1. clone the project in a folder called `io-onboarding-backend`
-2. go to the project's folder
-3. delete the files `cert.pem` and `key.pem` from the `certs` folder
-4. run the command `yarn generate-test-certs` to create SAML (SPID) certificates
-5. create a .env file and configure the values inside: `cp .env.example .env`
-6. run `docker-compose up -d` to start the containers
-7. edit your `/etc/hosts` file by adding:
-    ```
-    127.0.0.1    spid-testenv2
-    127.0.0.1    io-onboarding-backend
-    ```
-8. point your browser to [http://io-onboarding-backend:3000/metadata](http://io-onboarding-backend:3000/metadata) 
-and replace the content of the `testenv2/conf/sp_metadata.xml` file with the source of the page.
-9. in the `testenv2/conf/sp_metadata.xml` file:
-  - remove the `<EncryptionMethod>` elements
-  - between the elements `<KeyDescriptor>` and `<NameIDFormat>` add the following element:
-     ```
-     <SingleLogoutService 
-         Binding="urn:oasis:names:tc:SAML:2.0:bindings:HTTP-POST"
-         Location="http://io-onboarding-backend:3000/spid/logout" /> 
-    ``` 
-10. run `docker-compose restart` to restart the containers
-11. point your browser to [http://io-onboarding-backend:3000](http://io-onboarding-backend:3000)
-
-### SPID test login
-All the available SPID test users with their login credentials and attributes are listed in the page [http://spid-testenv2:8088/users](http://spid-testenv2:8088/users), you can add more if needed. 
-There are two default available users whose login credentials are respectively the following:
-  - username `pippo` and password `test`
-  - username `pinco` and password `test`
-
-In order to perform a SPID test login, point your browser to [http://io-onboarding-backend:3000/login?entityID=xx_testenv2&authLevel=SpidL2](http://io-onboarding-backend:3000/login?entityID=xx_testenv2&authLevel=SpidL2) and use the login credentials of one of the SPID test users. 
- 
-
-### Environment variables
-
-The table below describes all the Environment variables needed by the application.
+A local SPID test environment is needed to fully test the application features. [spid-testenv2](https://github.com/italia/spid-testenv2) has been selected to achieve this goal.
+
+### Build steps
+
+The *Dockerfile* and the `docker-compose.yaml` files used to build the application, are in the root of this repository.
+
+To build the application, run from the project root:
+
+```shell
+docker-compose up -d
+```
+
+>NOTE: the *docker-compose.yaml* file sets some environment variables that could be used to adapt the application features to specific needs. Variable values can be modified editing the *.env.example* file in this repository. More info about variables can be found in the dedicated paragraph, below in the readme.
```

Comment:
The file read by the application is _.env_: it must always be created from _.env.example_ before running the app, otherwise the app logs an error about missing required environment variables and exits.

Reply:
In my previous commit I changed the file name to .env.example in the docker-compose